### ISSUE TITLE
full access for OpenELEC settings even for non-privileged users

### ISSUE THREAD

**Issue by saljut7** [OpenELEC/service.openelec.settings#64]
System: RaspberryPi, Kodi with OpenELEC version 5.0.4, Confluence theme.

A limited user account (LOCKED: programs & script windows, file manager, settings "all", add-on manager | UNlocked music windows, videos window, pictures window) has full access to Settings / OpenELEC / Services:
- whole Samba Config including password in plaintext
- whole SSH Config (as far as I know "root" and "openelec" with no possibility to change)

Would it be possible to allow OpenELEC menu only for master-users?

**Comment by stefansaraev** [comment]
nope.

**Comment by saljut7** [comment]
@stefansaraev:

Thanks for your answer. So it is a problem of Kodi user configuration system to provide access to OpenELEC settings even for user who are not master-user?

**Comment by stefansaraev** [comment]
no. I say it's not going to be implemented in openelec, but as you have an rpi already, and it's nice educational device, you can start coding right now ;)

**Comment by saljut7** [comment]
Well, if my post sounded like a demand I'm sorry - that was not my intention. Just wanted to say that there is maybe a security problem. Someone at Kodi-forum advised me to post it here. 
I think bugtracing is a contribution as well (which doesn't mean that we speak about a bug here). If you say just "no" I'm fine and you can mark this "issue" solved. Who needs an explanations... ;)

**Comment by lsellens** [comment]
I know this is old but it was referenced in a suggested update on slack. I took a quick look to see how easily this could be accomplished. Honestly I think if you want to restrict any profile you should lock the programs and scripts section.
http://kodi.wiki/view/Profiles#The_profile_lock_preferences
as far as security problems are concerned openelec runs all of its addons as root which makes it not a very "secure" platform to begin with. Perhaps locking this section would hide some holes in a lower privileged profile for you. This would of course block launching any program addons. What you're suggesting isn’t impossible. I suppose we could check if "lock settings" is true on said profile and block access to the addon that way.
I'll make the change and pull request if the team would like me to. feedback?

**Comment by saljut7** [comment]
Thanks alot Isellens for the profile lock manual and pushing this topic. The point I didn't understand the time I opened this thread was: Why there are user profiles with a right management (so not only for different personal settings) if you can not set up something like a guest account with very limited access (just youtube) for example.
But I didn't want to complain. OpenELEC provides with Kodi the best entertainment system for free so there is nothing to complain about.

**Comment by michel3182** [comment]
![screenshot_20180718-005339](https://user-images.githubusercontent.com/30962459/43614465-de3b4b84-96b3-11e8-8aca-a47a2c6d6790.png)
